test/e2e/metrics_e2e_test.go

```diff
@@ -3,21 +3,22 @@
 package e2e
 
 import (
+	"bytes"
 	"context"
-	"fmt"
 	"regexp"
-	"time"
+	"strings"
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
+	io_prometheus_client "github.com/prometheus/client_model/go"
```

Comment:
nit: Is there a reason for the snake-case? AFAIK it's not conventional for import aliases.

Reply:
This package is actually generated by protobuf, so `io_prometheus_client` is its real name. I can sneak in a human-friendly import alias in a separate PR.